src-ts/tools/learn/course-details/CourseDetailsPage.tsx

```diff
@@ -57,16 +57,25 @@ const CourseDetailsPage: FC<CourseDetailsPageProps> = (props: CourseDetailsPageP
                                     dangerouslySetInnerHTML={{ __html: course.introCopy.join('<br /><br />') }}
                                 ></div>
 
-                                <h3 className='details'>Why should you complete this course?</h3>
+                                {progress?.status === 'completed' ? (
+                                    <>
+                                        <h3 className='details'>Suggested next steps</h3>
 
-                                <div className={styles['text']}>
-                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-                                    ut labore et dolore magna aliqua. Diam sit amet nisl suscipit. Ullamcorper sit amet risus nullam eget.
-                                    Etiam erat velit scelerisque in dictum non consectetur a. Mauris pellentesque pulvinar pellentesque
-                                    habitant morbi. Morbi enim nunc faucibus a pellentesque sit. Tortor pretium viverra suspendisse
-                                    potenti nullam ac. Ullamcorper malesuada proin libero nunc. Adipiscing elit duis tristique
-                                    sollicitudin nibh sit amet commodo. Orci a scelerisque purus semper eget duis at tellus at.
-                                </div>
+                                        <div className={styles['text']}>
+                                            <p>Now that you have completed the {course.title}, we’d recommend you enroll in another course to continue your learning. You can view our other courses from the Topcoder Academy course page. </p>
+                                        </div>
+                                    </>
+                                ) : (
+                                    <>
+                                        <h3 className='details'>Why should you complete this course?</h3>
+
+                                        <div className={styles['text']}>
+                                            <p>In this Responsive Web Design course, you'll learn the languages that developers use to build webpages: HTML (Hypertext Markup Language) for content, and CSS (Cascading Style Sheets) for design.</p>
```

Comment:
This text is specific to this course, so it can't be hardcoded into the page. We need to extract this to an attribute in the database, so please mock that up until I can add it to the DB and API.